host/xtest/regression_4000_data.h

```diff
@@ -8486,4 +8486,270 @@ static const uint8_t mac_cmac_vect12_out[] = {
 	0x69, 0x6a, 0x2c, 0x05, 0x6c, 0x31, 0x54, 0x10
 };
 
+/*
+ * TODO: AES CMAC and DES3 CMAC test vectors presented here are from the document
+ * which is obsolete as of October, 2016 (see https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-38b.pdf)
+ *
+ * We might want to use the current test vectors instead from
+ * https://csrc.nist.gov/projects/cryptographic-algorithm-validation-program/cavp-testing-block-cipher-modes
```

Comment:
I would prefer it too but it's OK to use older vectors too... as long as they are correct (but see my comment below)

Reply:
Thanks for the review.
Do you want me to remove this TODO?